VirtoCommerce.OrderModule.Test/OrderWorkflowControllerTest.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data.Common;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CacheManager.Core;
+using Microsoft.Practices.Unity;
+using Moq;
+using VirtoCommerce.Domain.Customer.Model;
+using VirtoCommerce.OrderModule.Core.Models;
+using VirtoCommerce.OrderModule.Core.Services;
+using VirtoCommerce.OrderModule.Data.Model;
+using VirtoCommerce.OrderModule.Data.Repositories;
+using VirtoCommerce.OrderModule.Data.Services;
+using VirtoCommerce.Platform.Core.Assets;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Data.Infrastructure;
+using VirtoCommerce.Platform.Data.Infrastructure.Interceptors;
+using VirtoCommerce.Platform.Testing.Bases;
+using Xunit;
+using Moq.Protected;
+
+
+
+namespace VirtoCommerce.OrderModule.Test
+{
+    [CLSCompliant(false)]
+    [Trait("OrderWorkflow", "OWF")]
+    public class OrderWorkflowControllerTest : TestBase
```

Comment:
The class name should be `OrderWorkflowServiceTest` since we're testing this service, not  `controller` which has difference meaning and purpose.